modules/apps/frontend-theme-locos/locos-theme/src/js/main.js

```diff
@@ -0,0 +1,63 @@
+
+AUI().ready(
+
+	/*
+  This function gets loaded when all the HTML, not including the portlets, is
+  loaded.
+  */
+
+  function() {
+  }
+);
+
+Liferay.Portlet.ready(
+
+  /*
+  This function gets loaded after each and every portlet on the page.
+
+  portletId: the current portlet's id
+  node: the Alloy Node object of the current portlet
+  */
+
+  function(portletId, node) {
+    // console.log(`Current portlet: ID${portletId} Object:${node}`);
+  }
+);
+
+Liferay.on(
+  'allPortletsReady',
+
+  /*
+  This function gets loaded when everything, including the portlets, is on
+  the page.
+  */
+
+  function() {
+
+    //add animated class to first row of layout
+    var childNodes = document.getElementById("main-content").childNodes;
```

Comment:
We can use jQuery instead of native JavaScript.  You can use the preloaded jQuery in Liferay via ```AUI.$()```.